docs/production/authentication/microsoft.mdx

```diff
@@ -181,6 +181,24 @@ You may need to restart your Mage server for the changes to take effect.
 
 </AccordionGroup>
 
+### Configurable environment variables
+To configure the environment for Microsoft Azure Cloud and Microsoft Azure Gov Cloud sign-in, 
+you need to set the following environment variables:
+1. You can specify the `AZURE_HOST` URLs based on your environment:
+```
+AzurePublicCloudHostUrl = "https://login.microsoftonline.com/";
+AzureChinaHostUrl = "https://login.chinacloudapi.cn/";
+AzureGermanyHostUrl = "https://login.microsoftonline.de/";
```

Comment:
@colefreeman Should the environment variable key be upper case snake case (like the env vars below) vs pascal case?

Reply:
Addressed and tested in mintlify dev env.